### DISCUSSION TITLE
docker run failed

### DISCUSSION THREAD

**Discussion by evehal** [mynaparrot/plugNmeet-server#217] [General]
docker run --rm -p 8080:8080 -v $PWD/config.yaml:/config.yaml mynaparrot/plugnmeet-server --config /config.yaml 
{"file":"/go/src/app/pkg/factory/mysqlDb.go:26","func":"github.com/mynaparrot/plugnmeet-server/pkg/factory.NewDbConnection","level":"fatal","msg":"dial tcp 127.0.0.1:3306: connect: connection refused","time":"2023-02-26T15:06:01Z"}

in config.yaml, i set mysql_info:
mysql_info:
  host: localhost
  port: 3306
  username: "root"
  password: "root123456"
  db: "plugnmeet"
  prefix: "pnm_"

the password is right,
why failed?

**Comment by jibon57** [comment]
Change the host with correct info. If that’s docket then either need host mood for MySQL to use as localhost.

**Reply by jibon57** [reply]
I think for your case will be host.docker.internal
